root/static/scripts/genre/components/GenreEditForm.js

```diff
@@ -26,17 +40,88 @@ type Props = {
   +form: GenreFormT,
 };
 
+/* eslint-disable flowtype/sort-keys */
+type ActionT =
+  | {+type: 'update-name', +action: NameActionT};
+/* eslint-enable flowtype/sort-keys */
+
+type StateT = {
+  +form: GenreFormT,
+  +guessCaseOptions: GuessCaseOptionsStateT,
+  +isGuessCaseOptionsOpen: boolean,
+  +previousForm?: GenreFormT | null,
```

Comment:
Is `previousName` used at all for genres?  I can only find references in root/static/scripts/alias/AliasEditForm.js.

Reply:
Nope. Whoops.